docs/hu/docs/index.md

```diff
@@ -0,0 +1,470 @@
+    * Mivel a `q` paraméter `= None`-al van deklarálva, ezért opcionális.
+    * `None` nélkül ez a mező kötelező lenne (mint például a body `PUT` kérések esetén).
+* a `/items/{item_id}` címre érkező `PUT` kérések esetén, a JSON-t a következőképpen olvassa be:
+    * Ellenőrzi hogy létezik a kötelező `name` nevű attribútum és `string`.
+    * Ellenőrzi hogy létezik a kötelező `price` nevű attribútum és `float`.
+    * Ellenőrzi hogy létezik a `is_offer` nevű opcionális paraméter, ami ha létezik akkor `bool`
+    * Ez ágyazott JSON objektumokkal is működik
+* JSONről való automatikus konvertálás.
+* dokumentáljuk mindent OpenAPI-al amit használható:
+    * Interaktív dokumentációs rendszerekkel.
+    * Automatikus kliens kód generáló a rendszerekkel, több nyelven.
+* Hozzá tartozik kettő interaktív dokumentációs web felület.
+
+---
+
+We just scratched the surface, but you already get the idea of how it all works.
+
+Próbáld kicserélni a következő sorban:
+
+```Python
+    return {"item_name": item.name, "item_id": item_id}
+```
+
+...ezt:
+
+```Python
+        ... "item_name": item.name ...
+```
+
+...err:
+
+```Python
+        ... "item_price": item.price ...
+```
+
+... És figyeld meg hogy a szerkesztőt automatikusan tudni fogja a típusokat és kiegészíti azokat:
+
+![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+Teljesebb példákért és funkciókért tekintsd meg a<a href="https://fastapi.tiangolo.com/tutorial/">Tutorial - User Guide</a>.
+
+**Spoiler veszélz**: a tutorial - user guidehoz tartozik:
+
+* **Paraméterek** deklarációja különböző helyekről: **header-ök**, **cookie-k**, **form mezők** és **fájlok**.
+* Hogyan állíts be **validációs feltételeket** mint a `maximum_length` vagy a `regex`.
+* Nagyon hatékony és erős **<abbr title="also known as components, resources, providers, services, injectables">Függőség Injekció</abbr>** rendszerek.
+* Biztonság és autentikáció beleértve, **OAuth2**, **JWT tokens** és **HTTP Basic** támogatást.
+* Több haladó (de ugyanannyira könnyű) technika **mélyen ágyazott JSON modellek deklarációjára** (Pydantic-nek köszönhetően).
+* **GraphQL** integráció <a href="https://strawberry.rocks" class="external-link" target="_blank">Strawberry</a>-vel és más könyvtárakkal.
+* több extra funkció (Starlette-nek köszönhetően) pl.:
+    * **WebSockets**
+    * rendkívül könnyű tesztek HTTPX és `pytest` alapokra építve
+    * **CORS**
+    * **Cookie Sessions**
+    * ...és több.
+
+## Teljesítmény
+
+Független TechEmpower benchmarkok azt mutatják, hogy alkalmazások amik **FastAPI**-t használnak és Uvicorn alatt futnak <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank"> hogy a **FastAPI** az egyik leggyorsabb az elérhető keretrendszerek közül</a>, csak a Starlette és Uvicorn mögött (amiket a FastAPI is használ). (*)
```

Comment:
Could we reword this a bit please?

Reply:
I couldn't find better wording for this. This is the original:

_Independent TechEmpower benchmarks show FastAPI applications running under Uvicorn as [one of the fastest Python frameworks available](https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7), only below Starlette and Uvicorn themselves (used internally by FastAPI). (*)_

Reply:
```A független TechEmpower benchmarkok szerint az Uvicorn alatt futó **FastAPI** alkalmazások az <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">egyik leggyorsabb Python keretrendszerek közé tartoznak</a>, éppen lemaradva a Starlette és az Uvicorn (melyeket a FastAPI belsőleg használ) mögött.
```

Reply:
Changed!